### PR TITLE
tutorial outline

### PR DESCRIPTION
**Pull request by moshez**

### CODE REVIEW

**Comment by mahmoud** [comment]
On a quick skim I like it! A couple structural thoughts:

* I think I'll keep the links to examples at the end as "further reading" or somesuch
* I think depending how big the implementation gets, I might split into two tutorials. Simple commands and commands with subcommands.

Way better than a blank slate though, much appreciated!

**Comment by moshez** [comment]
* Added the examples
* I haven't done the splitting, but it's already arranged into parts -- should be easy enough to split to multiple files later if this gets too long.

I suggest merging as-is, and then you can start filling in the blanks.

File: docs/tutorial.rst [hunk 1]
```diff
@@ -1,22 +1,219 @@
...
+is a quick but useful tool to cut recordings using
+``ffmpeg``.
+I use it to slice and dice the Python meetup recordings.
+It does not have subcommands or middleware,
+just a few flags.
+
+Glom
+~~~~
+
+`Glom`_
```
Comment (mahmoud) [inline docs/tutorial.rst:198]: are these links leading anywhere?

**Review by mahmoud — COMMENTED** [review]
looks good! Was about to merge, but had a question on the links?

File: docs/tutorial.rst (hunk 1 above)
Comment (moshez) [reply docs/tutorial.rst:201]: Good catch! fixed the links

**Pull request merged**